packages/api/src/index.ts

```diff
@@ -8,8 +8,9 @@ import fileUpload from "express-fileupload";
 import logger from "./utils/logger";
 import setupGlobalCustomMiddleware from "./middleware";
 dotenv.config({
-  path: path.resolve(process.cwd(), "/.env"),
+  path: path.resolve(process.cwd(), ".env"),
 });
+
 const PORT = process.env.PORT ?? 4000;
 
 // database connection
```

Comment:
add expose header like this    res.setHeader('Access-Control-Expose-Headers', 'x-access, x-refresh');